app/src/main/java/com/example/maintenancespace/ui/events/MaintenanceEventFragment.java

```diff
@@ -0,0 +1,73 @@
+package com.example.maintenancespace.ui.events;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.constraintlayout.widget.ConstraintLayout;
+import androidx.fragment.app.Fragment;
+
+import com.example.maintenancespace.R;
+import com.example.maintenancespace.databinding.FragmentMaintenanceEventBinding;
+import com.example.maintenancespace.models.events.MaintenanceEventModel;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class MaintenanceEventFragment extends Fragment {
+
+    private FragmentMaintenanceEventBinding binding;
+    private static final String EVENT_NAME = "EVENT_NAME";
+    private static final String EVENT_DESCRIPTION = "EVENT_DESCRIPTION";
+    private static final String EVENT_DATE = "EVENT_DATE";
+    private static final String EVENT_TIME = "EVENT_TIME";
+
+    public static final MaintenanceEventFragment newInstance(MaintenanceEventModel maintenanceEvent) {
+        MaintenanceEventFragment fragment = new MaintenanceEventFragment();
+
+        SimpleDateFormat dateFormatter = new SimpleDateFormat("M/d/yyyy");
+        SimpleDateFormat timeFormatter = new SimpleDateFormat("h:mma");
+        Date date = maintenanceEvent.getDate().toDate();
+        String dateFormatted = dateFormatter.format(date);
+        String timeFormatted = timeFormatter.format(date).toLowerCase();
+
+
+        Bundle bundle = new Bundle(2);
+        bundle.putString(EVENT_NAME, maintenanceEvent.getName());
+        bundle.putString(EVENT_DESCRIPTION, maintenanceEvent.getDescription());
+        bundle.putString(EVENT_DATE, dateFormatted);
+        bundle.putString(EVENT_TIME, timeFormatted);
+        fragment.setArguments(bundle);
+        return fragment ;
+    }
```

Comment:
This is needed to allow for re-rendering the fragment.  Constructors should be empty and unused in fragments.